src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+abstract class Figure implements Drawable, HasArea {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class definition.